Code/Tools/SceneAPI/SceneCore/Utilities/SceneGraphSelector.h

```diff
@@ -40,11 +48,30 @@ namespace AZ::SceneAPI::Utilities
         SCENE_CORE_API static bool IsMesh(const Containers::SceneGraph& graph, Containers::SceneGraph::NodeIndex& index);
         SCENE_CORE_API static bool IsMeshObject(const AZStd::shared_ptr<const DataTypes::IGraphObject>& object);
 
-        SCENE_CORE_API static Containers::SceneGraph::NodeIndex NoRemap(const Containers::SceneGraph& /*graph*/, const Containers::SceneGraph::NodeIndex& index)
+        SCENE_CORE_API static Containers::SceneGraph::NodeIndex NoRemap(
+            [[maybe_unused]] const Containers::SceneGraph& /*graph*/,
+            const Containers::SceneGraph::NodeIndex& index)
         {
             return index;
         }
-        SCENE_CORE_API static Containers::SceneGraph::NodeIndex RemapToOptimizedMesh(const Containers::SceneGraph& graph, const Containers::SceneGraph::NodeIndex& index);
+        SCENE_CORE_API static Containers::SceneGraph::NodeIndex RemapToOptimizedMesh(
+            const Containers::SceneGraph& graph,
+            const Containers::SceneGraph::NodeIndex& index);
```

Comment:
It would be nice to rename 'index' to 'unoptimizedMeshNodeIndex' or similar name to help understand the input parameter.